src/main/scala/org/holmesprocessing/totem/services/asnmeta/asnmeta.py

```diff
@@ -99,8 +105,11 @@ def __init__(self):
 
 def main():
     server = tornado.httpserver.HTTPServer(ASNApp())
-    server.listen(Config.settings.port)
-    tornado.ioloop.IOLoop.current().start()
+    server.listen(Config["settings"]["port"])
+    try:
+        tornado.ioloop.IOLoop.instance().start()
```

Comment:
I believe tornado perfers "tornado.ioloop.IOLoop.current().start()" over the instance() form. Can you confirm please?